app/scripts/acs/acs-graphs.js

```diff
@@ -18,8 +18,12 @@
         return module;
 
 
+        /**
+         * Replace text of length > max with an ellipsis
+         * Do no replacement if max === -1
+         */
         function addElipses(text, max) {
-            if (text.length > max) {
+            if (max !== -1 && text.length > max) {
```

Comment:
Feels very C (as in the programming language.) What is calling this function with negative values?